instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -58,7 +58,12 @@ def query(sql)
                 propagator.inject(sql, context: context)
               end
 
-              super
+              result = super
+
+              span.set_attribute('db.response.returned_rows', result.count)
+              span.set_attribute('db.response.affected_rows', result.affected_rows) unless result.affected_rows.nil?
```

Comment:
If you're adding more than one attribute you want to see the `add_attributes` interface for bulk setting attributes, otherwise you hit a mutex N many times, versus just the 1 time.
```suggestion
              attributes = { 'db.response.returned_rows' => result.count }
              attributes['db.response.affected_rows'] = result.affected_rows unless result.affected_rows.nil?
              span.add_attributes(attributes)
```